### PR TITLE
Float the done button on bottom of screen on long multi-select lists

### PR DESCRIPTION
**Pull request by isaacwilliams**
The "done" / "none of these" button floats on bottom of screen when the list is longer than the height of the screen

### CODE REVIEW

File: src/pages/BasePersonalisationPage.scss [hunk 1]
```diff
@@ -112,4 +112,24 @@
         text-align: center;
     }
 
+    .hasFloatingDoneButton {
+        padding-bottom: 80px;
+
+        .done-button {
+            @include riseIn(1s, 0s, forwards);
+
+            position: fixed;
```
Comment (danni) [inline src/pages/BasePersonalisationPage.scss:121]: Flexboxes would be better for this as they will push down as required. Have a look at how you do a sticky footer with flex boxes (already in our code).

File: src/pages/personalisation/BaseMultiQuestion.js [hunk 2]
```diff
@@ -118,11 +121,30 @@ class BaseMultiQuestion extends BaseQuestion {
...
     render(): ReactElement {
         let selected = this.selected;
 
         return (
-            <div className="BaseMultiQuestion">
+            <div
+                className={
+                    classnames(
+                        "BaseMultiQuestion",
+                        this.shouldRenderFloatingDoneButton()
```
Comment (danni) [inline src/pages/personalisation/BaseMultiQuestion.js]: `classnames` can take a class name and a truthy statement to evaluate: https://github.com/JedWatson/classnames

File: src/pages/BasePersonalisationPage.scss (hunk 1 above)
Comment (isaacwilliams) [inline src/pages/BasePersonalisationPage.scss:121]: I'm not sure that'll produce the correct behaviour - want to have it float over content, not be pushed down

Comment (danni) [inline src/pages/BasePersonalisationPage.scss:121]: Okay, if that's your intentional choice then disregard.

Comment (isaacwilliams) [inline src/pages/BasePersonalisationPage.scss:121]: I think it miiight be possible to do what I want using flex box but I can't find a good example

**Comment by DanielHeath** [comment]
Failed PR build is due to master having a bad commit

**Comment by DanielHeath** [comment]
++

**Pull request merged**